test/handler.test.js

```diff
@@ -195,6 +195,24 @@ describe('strong-error-handler', function() {
       });
     });
 
+    it('contains thrown message when debug=true', function(done) {
```

Comment:
agreed, and this will allow us to always have properties in desired order as you mentioned on previous message